mod/thewire/views/default/js/thewire.php

```diff
@@ -14,12 +14,10 @@
 		}
 	};
 
-	$("#thewire-textarea").live({
-		input: callback,
-		onpropertychange: callback
-	});
+	$(document).on('input', "#thewire-textarea", callback);
+	$(document).on('onpropertychange', '#thewire-textarea', callback);
```

Comment:
Thought that was strange too, buy just keeping it consistent with his the object was structured... Can add a separate fix if needed.